docs/ruleset_functions/get_proposed_hvac_modeled_with_virtual_heating.md

```diff
@@ -0,0 +1,40 @@
+# get_proposed_hvac_modeled_with_virtual_heating
+
+**Description:** Get the list of HeatingVentilationAirAconditioningSystem in which Appendix G Table G3.1 #10 c is applicable (i.e. space heating is modeled in the P_RMR but not the U_RMR).  Table G3.1 #10 c states that "where no heating system exists or no heating system has been submitted with design documents, the system type shall be the same system as modeled in the baseline building design and shall comply with but not exceed the requirements of Section 6."   
+
+**Inputs:**  
+- **U-RMR**: The U-RMR to determine if an HVAC system has been designed or is existing with heating.  
+- **P-RMR**: To determine if the same HVAC system has been modeled with heating in the P-RMR.  
+
+**Returns:**  
+- **proposed_hvac_modeled_with_virtual_heating_list_p**: A list that saves all the HVAC systems in which Appendix G Table G3.1 #10 c are applicable (i.e. modeled with virtual heating in the proposed).  
+ 
+**Function Call:**   
+
+1. match_data_element()  
+2. match_data_element_exist()
+
+## Logic:  
+- For each hvac_p in the the P_RMR: `for hvac_p in P_RMR...HeatingVentilationAirConditioningSystem:`       
+    - Reset has_virtual_heating_p boolean variable: `has_virtual_heating_p = FALSE`   
+        Below compares the heating systems associated with the hvac system across the P_RMR and U_RMR only if heating was modeled in the proposed. If heating was modeled in the proposed and NOT in the U_RMR (the heating system type is None in the U_RMR) then this means virtual heating was modeled in the proposed. We are checking the preheat coils because baseline systems 5 through 8 have preheat coils. 
```

Comment:
Please update to say as shown in square brackets.... "(the heating system type is None [or heating_system=Null] in the U_RMR )"